### DISCUSSION TITLE
[New Feature] hold backspace to erase characters in current message.

### DISCUSSION THREAD

**Discussion by Lylaktech** [Vinifera-Developers/Vinifera#1522] [Feature Requests]
### Description:
Currently in TS when writing a message in-game you need to tap the backspace key continuously to erase characters from it 1 by 1. What I would like to have is the ability to hold backspace to erase characters quickly.

**Comment by CCHyper** [comment]
Great QoL feature, I would be happy with this being the default behaviour. Any objections?
